src/App.jsx

```diff
@@ -59,7 +60,7 @@ const App = () => {
           <Route element={<ProtectedRoutes />}>
             <Route path='/chat' element={<JoinChat />} />
             <Route path='/chat/room' element={<ChatPage />} />
-            <Route path='/dashboard/users' element={<AdminManageUserPage />} />
```

Comment:
this` dashboard/users` should be protected , only the admin should access it

Reply:
Yes, done!

```jsx
<Route element={<AdminRoutes />}>
      <Route path='/dashboard/users' element={<AdminManageUserPage />} />
</Route>
```

<img width="1440" alt="Screenshot 2023-06-22 at 11 51 01" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/9588f53d-f2b4-4470-987e-dde8d7ded8c8">